internal/help/help.go

```diff
@@ -0,0 +1,85 @@
+package help
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/nestoca/joy/internal/config"
+)
+
+const allCommandsKey = "_all"
```

Comment:
This kills me to have to nit this again. But can the prefix be any symbol that's not considered a word character? Word characters are `a-zA-Z_` also I would consider a `-` to not be a good candidate for this. Probably same for `$` but less so.  

`!` | `@` | `#` | `&` | `*` | `~` would all be fantastic choices.

Reply:
EDIT: `&` and `*` would be poor choices since those are anchor tags in yaml documents